spec/02-integration/09-hybrid_mode/09-node-id-persistence_spec.lua

```diff
@@ -87,7 +88,7 @@ for _, v in ipairs({ {"off", "off"}, {"on", "off"}, {"on", "on"}, }) do
   local rpc, rpc_sync = v[1], v[2]
 
 for _, strategy in helpers.each_strategy() do
-  describe("node id persistence " .. " rpc_sync=" .. rpc_sync, function()
+  describe("node id persistence rpc_sync = " .. rpc_sync, function()
```

Comment:
Do we need this change? I think that it is not necessary for this PR.

Reply:
just merge this tiny fix from ee